### PR TITLE
Removes usage of make

### PR DESCRIPTION
**Pull request by ckartik**
* Replaces make settlemnt with docker command
* Removes redundant db-clean for settlement layer as we wipe all docker details

### CODE REVIEW

**Review by kant777 — APPROVED** [review]

File: mev-commit-cli.sh [hunk 1]
```diff
@@ -65,7 +65,12 @@ NEXT_PUBLIC_WALLET_CONNECT_ID=
 DD_KEY=${datadog_key}
 EOF
 
-    make -C "$GETH_POA_PATH"/geth-poa up-prod-settlement
+    # Run settlement layer
+    # Set environment variables
+    export AGENT_BASE_IMAGE=nil
+    export L2_NODE_URL=nil
+    # Run Docker Compose
+    docker compose --profile settlement -f "$GETH_POA_PATH/geth-poa/docker-compose.yml" up -d --build
```
Comment (shaspitz) [inline mev-commit-cli.sh:73]: We should deploy bridge components as well, pattern matching `make up-prod-build`

File: mev-commit-cli.sh [hunk 2]
```diff
@@ -65,7 +65,12 @@ NEXT_PUBLIC_WALLET_CONNECT_ID=
 DD_KEY=${datadog_key}
 EOF
 
-    make -C "$GETH_POA_PATH"/geth-poa up-prod-settlement
+    # Run settlement layer
+    # Set environment variables
+    export AGENT_BASE_IMAGE=nil
```
Comment (shaspitz) [inline mev-commit-cli.sh:70]: This should have value of `shaspitz/hyperlane-agent-mac:6985064` for spinning up settlement layer locally on mac, or `gcr.io/abacus-labs-dev/hyperlane-agent@sha256:854f92966eac6b49e5132e152cc58168ecdddc76c2d390e657b81bdaf1396af0` for amd

Comment (shaspitz) [reply mev-commit-cli.sh:70]: Can make this a flag or param

**Comment by shaspitz** [comment]
will address my comments in other PR

**Pull request merged**